src/Console/Report/FixReport/ReportSummary.php

```diff
@@ -72,7 +72,7 @@ public function isDryRun(): bool
     }
 
     /**
-     * @return array<string, array{appliedFixers: list<string>, diff: string}>
+     * @return array<string, array{appliedFixers: list<string>, diff: string, extraInfoFixers: array{helpUri: array<string, string>}|array{}}>
```

Comment:
Why not:

```suggestion
     * @return array<string, array{appliedFixers: list<string>, diff: string, helpUris: null|array<string, string>}>
```

or 

```suggestion
     * @return array<string, array{appliedFixers: list<string>, diff: string, helpUris?: array<string, string>}>
```

The first one make the `helpUri` nullable, the second one makes it optional. In both cases we can detect if there's info with URIs or not. I also find `extraInfoFixers` superfluous, I would rather add one level field like `helpUris` (plural, as it's a collection).

In fact, I would like to do `appliedFixers: list<array{name: string, helpUri: string}>`, but `Runner` also is not internal so we can't change the format of the returned array. Adding new field should be OK, though.

@keradus please let me know what you think 🙂.

Reply:
@Wirone Stop wasting your time with this current PR. Even if code is almost ready, I should change the `extraInfoFixers`.

I'll give more explains when my new version will be ready. I hope to propose something in next hours (now, I'm back)